src/components/DopplerPlus/PlanCalculator.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+
+const Slider = ({ min, max, step, defaultValue, handleChange }) => {
```

Comment:
I think this slider perhaps should be in a different file or allowed to be exported from this file. Don't you think?

Reply:
Yes, I thought about it, but since we use it only in this element, maybe it's not necessary. We do something similar in other elements. I don't know what is the rule that makes to keep an element in the same file or make it exportable.